lib/slop/options.rb

```diff
@@ -26,7 +26,7 @@ class Options
     def initialize(**config)
       @options    = []
       @separators = []
-      @banner     = "usage: #{$0} [options]"
+      @banner     = config[:banner] ? config[:banner] : "usage: #{$0} [options]"
```

Comment:
Ah good point! I missed your detail of that previously. Well this is up to you, but as it currently does  support that I think we should keep support for it!